frontend/src/js/external-forms/form-components/DropzoneBetweenElements.tsx

```diff
@@ -5,52 +5,41 @@ import Dropzone, {
   PossibleDroppableObject,
 } from "../../ui-components/Dropzone";
 
-interface Props {
-  onDrop: (props: PossibleDroppableObject, monitor: DropTargetMonitor) => void;
-  acceptedDropTypes: string[];
-  top: number;
-  height: number;
-}
-
-const Root = styled("div")`
-  display: flex;
-  height: 4px;
-`;
+const LINE_HEIGHT = 4;
 
 const Line = styled("div")`
   background-color: ${({ theme }) => theme.col.blueGrayDark};
-  margin: 1px 0;
-  height: 4px;
-  border-radius: 2px;
-  flex-grow: 1;
+  height: ${LINE_HEIGHT}px;
+  width: 100%;
+  border-radius: ${({ theme }) => theme.borderRadius};
 `;
 
-const SxDropzone = styled(Dropzone)<{ height: number; top: number }>`
-  height: ${({ height }) => height}px;
-  top: ${({ top }) => top}px;
+const SxDropzone = styled(Dropzone)`
+  height: 30px;
   position: absolute;
+  top: 0;
+  left: 0;
+  transform: translateY(calc(-50% - ${LINE_HEIGHT / 2}px));
```

Comment:
No need to allow passing height or top, we can simply set height to 30px, that makes the droppable area always 30px high. Then we can use `transform` to move the dropzone "between" the elements by `-50%` related to its own height of 30px, minus half the height of the line. That makes it sit right in the middle

Reply:
Yeah good point, fixed it with a classname and making the last dropzone less high